Requirement.txt

Aryan-Prakhar opened the pull request:
Added a requirement.txt file for control.py of EOG_DinoGame so as to directly install all libraries required to run Dino game.

Aryan-Prakhar committed:
Update README.md

Aryan-Prakhar committed:
Added a requirement.txt file of control.py

lorforlinux commented on .gitignore:
```diff
@@ -30,3 +30,4 @@
 *.exe
 *.out
 *.app
+6_EOGDinoGame/venv/
```
This should not be specific to a project folder, please make it generic.

Aryan-Prakhar replied:
I have made the updates regarding this. Please review.

Aryan-Prakhar committed:
Added a python library file
 This file will automatically connect with the Arduino serial port interface with any any change in code

Aryan-Prakhar committed:
Update .gitignore

Removed Dino Game virtual environment

lorforlinux commented:
@Aryan-Prakhar Thank you so much for your contribution, I am merging this now!
